sa/sa.go

```diff
@@ -473,40 +532,55 @@ func (ssa *SQLStorageAuthority) MarkCertificateRevoked(ctx context.Context, seri
 
 	tx, err := ssa.dbMap.Begin()
 	if err != nil {
-		return
+		return err
 	}
 
-	var status core.CertificateStatus
-	err = tx.SelectOne(
-		&status,
-		fmt.Sprintf("SELECT %s FROM certificateStatus WHERE serial = ?", CertificateStatusFields),
-		serial,
-	)
-	if err == sql.ErrNoRows {
-		return Rollback(tx, fmt.Errorf("No certificate status found for %s", serial))
+	var statusObj interface{}
+
+	if features.Enabled(features.CertStatusOptimizationsMigrated) {
+		statusObj, err = tx.Get(certStatusModelv2{}, serial)
```

Comment:
I think this accidentally undoes Roland's recent work to move to explicit `SELECT`.